tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+minversion = 3.9.0
+envlist =
+    lint
+    py{36,37,38}
+
+# do not enable skip missing to avoid CI false positives
+skip_missing_interpreters = False
```

Comment:
Won't this mean that it won't work for users without all three versions of Python installed?

Reply:
There is a long story about this, I will find some links tomorrow. All openstack projects have this added http://codesearch.openstack.org/?q=skip_missing_interpreters&i=nope&files=tox.ini&repos= -- with extra comment to avoid accidental regressions.

Shortly we decided to enforce it to this value because it happened multiple times to get some faile positive that nobody notices for long time.

You can blame tox for this but if you set this to true and run `tox -e py38` on a machine that does not have python3.8 already installed, you will get a success exit code!

As you can expect, the downside is that running just "tox" may no longer return 0 if you do not have all supported versions of python installed. I do not remember seeing complaints, especially after they understood the dangers.

Reply:
The purpose of using tox was to ease local testing, not necessarily for CI. To that end, I think we should allow `tox` to run without the `-e` switch, even if the user doesn't have three version of Python.